cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/SimpleGeometryCoverage.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright (c) 1998-2018 John Caron and University Corporation for Atmospheric Research/Unidata
+ * See LICENSE for license information.
+ */
+package ucar.nc2.ft2.coverage.simpgeometry;
+
+import ucar.ma2.DataType;
+import ucar.ma2.InvalidRangeException;
+import ucar.ma2.IsMissingEvaluator;
+import ucar.ma2.Section;
+import ucar.nc2.Attribute;
+import ucar.nc2.AttributeContainerHelper;
+import ucar.nc2.Dimension;
+import ucar.nc2.VariableSimpleIF;
+import ucar.nc2.dataset.CoordinateAxis;
+import ucar.nc2.util.Indent;
+import ucar.nc2.ft2.coverage.simpgeometry.CFGEOMETRY;
+import ucar.nc2.ft2.coverage.CoverageReader;
+import ucar.nc2.ft2.coverage.SubsetParams;
+import ucar.nc2.ft2.coverage.adapter.SimpleGeometryCS;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Formatter;
+import java.util.List;
+
+/**
+ * SimpleGeometry - forked from Coverage.java
+ * Immutable after setCoordSys() is called.
+ *
+ * @author Katie
+ * @author Carron
+ * @since 8/13/2018
+ */
+// @Immutable
+public class SimpleGeometryCoverage implements VariableSimpleIF, IsMissingEvaluator { //add interface with geometry info? //subsetting data will be diffferent?? is it subset now

```

Comment:
The acceptance criteria require that there be tests for this class. Could some tests be created?